items-service/src/main/java/com/spring/utils/ExceptionUtils.java

```diff
@@ -0,0 +1,17 @@
+package com.spring.utils;
+
+import com.spring.model.AppError;
+
+public class ExceptionUtils extends RuntimeException {
+    private AppError error;
```

Comment:
add a getter for this so that you can call this object from @ControllerAdvice class while handling the exception.